Makefile

```diff
@@ -345,6 +351,12 @@ deploy-postgres: deploy-namespace
 	python3 ./tools/deploy_postgres.py --namespace "$(NAMESPACE)" --target "$(TARGET)" \
 		--apply-manifest $(APPLY_MANIFEST) --persistent-storage $(PERSISTENT_STORAGE)
 
+deploy-kafka: deploy-namespace
+	python3 ./tools/deploy_kafka.py --namespace "$(NAMESPACE)" --target "$(TARGET)" \
+		--apply-manifest $(APPLY_MANIFEST)
+
+deploy-all-event-streaming: deploy-kafka deploy-all
```

Comment:
I want to limit the amount of makefile targets if that's possible
Can we deploy kafka as part of ``deploy-all`` based on the value of ``ENABLE_EVENT_STREAMING``?